<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Sample Test Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="Failed_Case.RetryListenerClass"/>
    <listener class-name="Failed_Case.RetryListenerClass"/>
  </listeners>
  <test thread-count="5" name="VerifycurrentUrl()(failed)">
    <classes>
      <class name="Failed_Case1.TestNGExampleTests">
        <methods>
          <include name="VerifyCurrentURl"/>
          <include name="setup"/>
        </methods>
      </class> <!-- Failed_Case1.TestNGExampleTests -->
    </classes>
  </test> <!-- VerifycurrentUrl()(failed) -->
  <test thread-count="5" name="VerifyText()(failed)">
    <classes>
      <class name="Failed_Case1.TestNGExampleTests">
        <methods>
          <include name="VerifyCurrentURl"/>
          <include name="verifycurrentUrl"/>
          <include name="setup"/>
        </methods>
      </class> <!-- Failed_Case1.TestNGExampleTests -->
    </classes>
  </test> <!-- VerifyText()(failed) -->
</suite> <!-- Failed suite [Sample Test Suite] -->
